awx/ui_next/src/screens/Template/shared/WorkflowJobTemplateForm.jsx

```diff
@@ -0,0 +1,176 @@
+  handleSubmit,
+  handleCancel,
+  i18n,
+  template = {},
+}) {
+  const [contentError, setContentError] = useState(null);
+  const [inventory, setInventory] = useState(
+    template?.summary_fields?.inventory || null
+  );
+  const [organization, setOrganization] = useState(
+    template?.summary_fields?.organization || null
+  );
+
+  if (contentError) {
+    return <ContentError error={contentError} />;
+  }
+  return (
+    <Formik
+      onSubmit={handleSubmit}
+      initialValues={{
+        name: template.name || '',
+        description: template.description || '',
+        inventory: template.summary_fields?.inventory?.id || '',
+        organization: template.summary_fields?.organization?.id || '',
+        labels: template.summary_fields?.labels?.results || [],
+        variables: template.variables || '---',
+        limit: template.limit || '',
+        scmBranch: template.scm_branch || '',
+        ask_limit_on_launch: template.ask_limit_on_launch || false,
+        ask_inventory_on_launch: template.ask_inventory_on_launch || false,
+        ask_variables_on_launch: template.ask_variables_on_launch || false,
+        ask_scm_branch_on_launch: template.ask_scm_branch_on_launch || false,
+      }}
+    >
+      {formik => (
+        <Form autoComplete="off" onSubmit={formik.handleSubmit}>
+          <FormRow>
+            <FormField
+              id="wfjt-name"
+              name="name"
+              type="text"
+              label={i18n._(t`Name`)}
+              validate={required(null, i18n)}
+              isRequired
+            />
+            <FormField
+              id="wfjt-description"
+              name="description"
+              type="text"
+              label={i18n._(t`Description`)}
+            />
+            <Field
+              id="wfjt-organization"
+              label={i18n._(t`Organization`)}
+              name="organization"
+            >
+              {({ form }) => (
+                <OrganizationLookup
+                  helperTextInvalid={form.errors.organization}
+                  isValid={
+                    !form.touched.organization || !form.errors.organization
+                  }
+                  onBlur={() => form.setFieldTouched('organization')}
+                  onChange={value => {
+                    form.setFieldValue('organization', value?.id || null);
+                    setOrganization(value);
+                  }}
+                  value={organization}
+                  touched={form.touched.organization}
+                  error={form.errors.organization}
+                />
+              )}
+            </Field>
+          </FormRow>
+          <FormRow>
+            <Field name="inventory">
+              {({ form }) => (
+                <InventoryLookup
+                  value={inventory}
+                  onBlur={() => form.setFieldTouched('inventory')}
+                  tooltip={i18n._(t`Select the inventory containing the hosts
```

Comment:
We probably want this tooltip to say something about the fact that the inventory will override the inventory for all nodes.

<img width="480" alt="Screen Shot 2020-02-12 at 10 41 16 AM" src="https://user-images.githubusercontent.com/9889020/74350820-4c56cc00-4d84-11ea-8065-54a1139a0c3a.png">

When in doubt, I'd match up the tooltips for form fields with whatever we have in the existing app.

Reply:
👍